adafruit_circuitplayground/bluefruit.py

```diff
@@ -161,7 +161,9 @@ def play_mp3(self, file_name):
             with self._audio_out(
                 board.SPEAKER
             ) as audio:  # pylint: disable=not-callable
-                mp3file = audiomp3.MP3Decoder(open(file_name, "rb"))
+                mp3file = audiomp3.MP3Decoder(
+                    open(file_name, "rb")  # pylint: disable=consider-using-with
```

Comment:
Looks good--Just curious why are we not using  `with open(...)` here?

Reply:
I didn't realise you could do nested `with`s! Fixing now.